src/components/theme/ConfirmTheme.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <section class="content-pop flex-col flex-cc">
```

Comment:
why confirm theme component has these content? I think these only in retro item component.